core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/JDBCSourceAsColumnarStore.scala

```diff
@@ -179,6 +290,10 @@ class JDBCSourceAsColumnarStore(override val connProperties: ConnectionPropertie
           stmt.close()
         } catch {
           // TODO: test this code
+          //TODO:Suranjan This code is no needed now.
+          // Need to rollback.
```

Comment:
When you will add rollback. I think another PR is there. Are we doing rollback only in case of NonFatal exception or otherwise too?